protos/profiles.proto

```diff
@@ -20,24 +20,24 @@ message Profile {
 }
 
 message JDBC {
-  string host = 2;
-  int64 port = 3;
-  string user = 4;
-  string password = 5;
-  string database = 6;
+  required string host = 2;
```

Comment:
As we are using proto3 syntax, I don't think this is valid. On the typescript side, Protobuf.js compiler doesn't care I imagine, but for other languages using the proper proto compiler I suspect this will fail.

You can check by running:
```
bazel build //protos:java
```
The above should fail.